docker/utils/utils.py

```diff
@@ -980,6 +984,11 @@ def create_container_config(
             'stop_signal was only introduced in API version 1.21'
         )
 
+    if healthcheck is not None and compare_version('1.24', version) < 0:
+        raise errors.InvalidVersion(
+            'Health options were only introduced in API version 1.24'
+        )
+
```

Comment:
You can use the `host_config_version_error` utility function here instead of writing a custom error string.